tools/dreport.d/plugins.d/journalpid

```diff
@@ -1,20 +1,13 @@
 #!/bin/bash
```

Comment:
Enable journalpretty for elog/core is going to meet the requirement right? incase you are not using journal-pid you can remove from down stream. 
Here yu are keepin two plugins with same name. that is not good.

Reply:
ok